pkg/cmd/pr/list/list.go

```diff
@@ -116,6 +118,19 @@ func listRun(opts *ListOptions) error {
 		return err
 	}
 
+	argv, err := shlex.Split(opts.Search)
+	if err != nil {
+		return err
+	}
+
+	re := regexp.MustCompile(`^merged:.*`)
```

Comment:
Nitpick: you don't need to end the regexp with `.*`. It will match the prefix nonetheless.

Even better: if the prefix is static string, you can use `strings.HasPrefix`